monorepo/website/src/pages/docs/how-to/edit-group.mdx

```diff
@@ -0,0 +1,17 @@
+---
+layout: ../../../layouts/DocLayout.astro
+title: "Editing group information"
+order: 22
+---
+import CautionBox from '../../../components/asides/CautionBox.astro';
+
+Any member of a group can edit group information, which means that you can edit group information for all the groups that you are a member of.
+
+1. Log into your account and enter your account page by clicking 'My account' in the top-right corner of the webpage
+2. Under 'My groups' click the group that you want to edit
+3. In the group detail view, click the 'Edit group' button in the top right corner
+4. Edit any property of the group you want to change and click 'Update group' to save your changes
+
+<CautionBox>
+If you change the name of a group the center name in corresponding ENA submissions will also change.
```

Comment:
(cool PR) This implies to me that the change will propagate to existing submissions which I would guess it won't

Reply:
Uhm, I think I'm not too familiar with how this ENA stuff works, I'd ask @corneliusroemer to weigh in? We have the same line of text in the loculus docs, and there Cornelius was part of coming up with the text.

Reply:
I've added a suggestion